src/Maestro/tests/Scenarios/githubflow-nonbatched-all-checks-successful.ps1

```diff
@@ -63,8 +63,8 @@ try {
     # Add the foo and bar dependencies
     try {
         Push-Location -Path $(Get-Repo-Location $targetRepoName)
-        Darc-Command add-dependency --name 'Foo' --type product --repo $sourceRepoUri
-        Darc-Command add-dependency --name 'Bar' --type product --repo $sourceRepoUri
+        Darc-Command @( "add-dependency", "--name", "Foo", "--type product", "--repo", "$sourceRepoUri" )
```

Comment:
```suggestion
        Darc-Command @( "add-dependency", "--name", "Foo", "--type", "product", "--repo", "$sourceRepoUri" )
```

All the github tests are failing when adding dependencies due to a similar bug. Makes me think this approach is a little brittle and cumbersome to write? Not sure if I have a better suggestion yet though.